tests/core/config/checkers/test_config_id_checker.py

```diff
@@ -23,7 +23,7 @@ def test_check_standalone_mode(self, caplog):
         assert len(Config._collector.errors) == 0
 
         Config.configure_data_node(id="foo", storage_type="in_memory")
-        Config.configure_scenario(id="bar", pipeline_configs=[])
+        Config.configure_scenario(id="bar", task_and_data_node_configs=[])
```

Comment:
THe signature is wrong, no?

Reply:
yes, it's the old version I haven't fixed yet